utils/generate_api.py

```diff
@@ -590,9 +590,15 @@ def read_modules() -> Any:
             parts_new = {}
 
             for m in params:  # pylint: disable=invalid-name
-                a = dict(  # pylint: disable=invalid-name
-                    type=m["schema"]["type"], description=m["description"]
-                )  # pylint: disable=invalid-name
+                if "description" in m:
+                    a = dict(  # pylint: disable=invalid-name
+                        type=m["schema"]["type"], description=m["description"]
+                    )  # pylint: disable=invalid-name
+                else:
+                    a = dict(  # pylint: disable=invalid-name
+                        type=m["schema"]["type"]
+                    )  # pylint: disable=invalid-name
+
```

Comment:
@AbitraryYu, could you handle [params description](https://github.com/opensearch-project/opensearch-py/blob/main/utils/generate_api.py#L593) similarly to how parts description is handled [here](https://github.com/opensearch-project/opensearch-py/blob/main/utils/generate_api.py#L630)?

```
                a = dict(type=m["schema"]["type"])  # pylint: disable=invalid-name

                if "description" in m:
                    a.update({"description": m["description"]})

```